lms/templates/courseware/course_navigation-sneak-peek-warning.html

```diff
@@ -13,11 +13,15 @@
     reg_url = reverse('course-specific-register', args=[course.id.to_deprecated_string()])
   else:
     reg_url = reverse('register_user')
+  tos_link = u"<a href='{}'>".format(marketing_link('TOS'))
+  privacy_link = u"<a href='{}#privacy'>".format(marketing_link('TOS'))
   %>
-   ${_("Non-registered mode.  {tag_start}Register{tag_end} to save your course progress.")\
+   ${_("Non-registered mode.  {tag_start}Register{tag_end} to save your course progress. By exploring the course, you are agreeing to our {tos_link_start}Terms of Service{tag_end} and {privacy_link_start}Privacy Policy{tag_end}. Please read them carefully.")\
```

Comment:
The other benefit to splitting this onto another line is that we can resuse the existing template fragment for the second sentence.

In a number of places, we only ever display the "By exploring the course, ... Please read them carefully." bit.
We can include it here via `course_info_extra_sneakpeek.html`

As is, we need to edit